src/java_tools/junitrunner/java/com/google/testing/junit/runner/BazelTestRunner.java

```diff
@@ -79,7 +79,15 @@ public static void main(String[] args) {
       System.exit(2);
     }
 
-    int exitCode = runTestsInSuite(suiteClassName, args);
+    int exitCode;
+    try {
+      exitCode = runTestsInSuite(suiteClassName, args);
+    } catch (Throwable e) {
+      // An exception was thrown by the runner. Print the error to the output stream so it will be logged
+      // by the executing strategy, and return a failure, so this process can gracefully shut down.
+      e.printStackTrace(System.err);
```

Comment:
@irengrig Awesome, will do!
Do you think returning `1` is the right thing here? Perhaps error code `2` makes more sense, since this this is an unexpected termination?

Reply:
Yep, I was also thinking if we better return some code like 12345, not 1.
But just printing the error is good enough.